### PR TITLE
Tron 18 a

### PR DESCRIPTION
**Pull request by muhammad-ahmad-shahbaz**
TICKET: https://pakw.atlassian.net/browse/TRON-18

**What?**

We want to create a migration to create a UserAdd table and its respective model where we will store the users posted adds data.

**Acceptance Criteria:**

Add a migration to create UserAdd table in the database.

Add following columns with their respective types in it which are as follows:

    city (string) (required) not null

    registered in (string)

    exterior color (string) (required) not null

    mileage (float) (required) not null

    price (integer) (required) not null

    add description (string) (required) not null

    mobile number (integer) (required) not null

    secondary number (integer)

Make sure to add schema level validation for these columns.

Once the migration will be added create its respective model and add model level validations for the above columns as well.

Make its relationship with User model. (Each user can have multiple adds and each add belongs to a specific user)

Use Active Storage to attach images to the UserAdd model.

### CODE REVIEW

**Comment by Adil449** [comment]
@muhammad-ahmad-shahbaz Still too many files, also please add the PR description in proper bullet points.

**Comment by muhammad-ahmad-shahbaz** [comment]
fixed those issues

File: app/models/user.rb [hunk 1]
```diff
@@ -1,12 +1,11 @@
-class User < ApplicationRecord
+class User < ApplicationRecord
```
Comment (Adil449) [inline app/models/user.rb:1]: Remove trailing white space here

File: app/models/user.rb [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
+class User < ApplicationRecord  
   devise :database_authenticatable, :registerable,
             :recoverable, :rememberable, :validatable
   has_one_attached :profile_picture
   validates :full_name, :gender, :country, :city, :username, presence: true
   validates :username, uniqueness: true
   validates :price, presence: true, 
             format: { with: /\A\d+(?:\.\d{0,2})?\z/ }, numericality: { greater_than: 0, less_than: 1000000 }
-
-  validates :password, :password_confirmation, presence: true
```
Comment (Adil449) [inline app/models/user.rb:10]: why this change is showing up here in the pull request?

File: app/models/user.rb (hunk 1 above)
Comment (muhammad-ahmad-shahbaz) [reply app/models/user.rb:1]: ok

File: app/models/user.rb (hunk 2 above)
Comment (muhammad-ahmad-shahbaz) [reply app/models/user.rb:10]: got it

**Review by Adil449 — APPROVED** [review]
